raiden-ts/src/transport/epics.ts

```diff
@@ -420,18 +451,14 @@ function fetchSortedMatrixServers$(matrixServerLookup: string, httpTimeout: numb
  * @param deps - RaidenEpicDeps-like/partial object
  * @param deps.address - Our address (to compose matrix user)
  * @param deps.signer - Signer to be used to sign password and displayName
+ * @param caps - Transport capabilities to set in user's avatar_url
  * @returns Observable of one { matrix, server, setup } object
  */
 function setupMatrixClient$(
   server: string,
   setup: RaidenMatrixSetup | undefined,
-  {
-    address,
-    signer,
-  }: {
-    address: RaidenEpicDeps['address'];
-    signer: RaidenEpicDeps['signer'];
-  },
+  { address, signer }: Pick<RaidenEpicDeps, 'address' | 'signer'>,
```

Comment:
What is the `Pick` type?

Reply:
It's a typescript type utility which selects the given keys from the interface passed as parameter; in this case, it'll be like `{ address: RaidenEpicDeps['address']; signer: RaidenEpicDeps['signer'] }`. I use it to avoid requiring the whole `RaidenEpicDeps` object on local functions/utils, so I can pass this destructured subset